changelog/unreleased/kong/wasm-dispatch-dns.yml

```diff
@@ -0,0 +1,5 @@
+message: >
+  **proxy-wasm**: The Wasm subsystem will reuse the global `kong.dns` DNS
```

Comment:
Nit: if it's a bug, we should describe the behavior it fixes as seen by the user. Otherwise, we could call it a 'refactor' instead.

Reply:
This is good feedback, thanks. There's no changelog type for generic refactor/chore items--probably by design, I guess. I'll just remove the changelog entry for this since it doesn't have any user-facing effects.